metricflow/test/snapshots/test_dataflow_to_sql_plan.py/SqlQueryPlan/Postgres/test_conversion_metric__plan0_optimized.sql

```diff
@@ -0,0 +1,87 @@
+-- Compute Metrics via Expressions
+SELECT
+  visit__referrer_id
+  , CAST(buys AS DOUBLE PRECISION) / CAST(NULLIF(visits, 0) AS DOUBLE PRECISION) AS visit_buy_conversion_rate
+FROM (
+  -- Combine Aggregated Outputs
+  SELECT
+    COALESCE(subq_18.visit__referrer_id, subq_28.visit__referrer_id) AS visit__referrer_id
+    , MAX(subq_18.visits) AS visits
+    , MAX(subq_28.buys) AS buys
+  FROM (
+    -- Aggregate Measures
+    SELECT
+      visit__referrer_id
+      , SUM(visits) AS visits
+    FROM (
+      -- Read Elements From Semantic Model 'visits_source'
+      -- Metric Time Dimension 'ds'
+      -- Pass Only Elements:
+      --   ['visits', 'visit__referrer_id']
+      SELECT
+        referrer_id AS visit__referrer_id
+        , 1 AS visits
+      FROM ***************************.fct_visits visits_source_src_10011
+    ) subq_17
+    GROUP BY
+      visit__referrer_id
+  ) subq_18
+  FULL OUTER JOIN (
+    -- Find conversions for EntitySpec(element_name='user', entity_links=()) within the range of count=7 granularity=TimeGranularity.DAY
+    -- Pass Only Elements:
+    --   ['buys', 'visit__referrer_id']
+    -- Aggregate Measures
+    SELECT
+      visit__referrer_id
+      , SUM(buys) AS buys
+    FROM (
+      -- Dedupe the fanout on (MetadataSpec(element_name='mf_internal_uuid'),) in the conversion data set
```

Comment:
Same with any other SQL comments, we should remove any class references and use a name they can recognize in the SQL

Reply:
Leaving this unresolved, will do this once we resolve the other reviews!